app/config/locale/translations/zh-tw.json

```
@@ -111,117 +117,117 @@
     "countries.iq": "伊拉克",
     "countries.is": "冰島",
     "countries.il": "以色列",
-    "countries.it": "意大利",
+    "countries.it": "義大利",
     "countries.jm": "牙買加",
     "countries.jo": "約旦",
     "countries.jp": "日本",
-    "countries.kz": "哈薩克斯坦",
-    "countries.ke": "肯尼亞",
-    "countries.kg": "吉爾吉斯斯坦",
+    "countries.kz": "哈薩克",
+    "countries.ke": "肯亞",
+    "countries.kg": "吉爾吉斯",
     "countries.kh": "柬埔寨",
-    "countries.ki": "基裏巴斯",
-    "countries.kn": "聖基茨和尼維斯",
+    "countries.ki": "吉里巴斯",
+    "countries.kn": "聖克里斯多福及尼維斯",
     "countries.kr": "韓國",
     "countries.kw": "科威特",
-    "countries.la": "老撾",
+    "countries.la": "寮國",
```

```suggestion
    "countries.la": "寮人",
```

Source: https://www.mofa.gov.tw/CountryInfo.aspx?CASN=0&n=487&sms=33&s=157

---

The full name is 寮人民民主共和國. But we don't call it 寮人. You can check the tabs in that page.

---

彳亍。只要符合你们口语化用法就可以了。